README.md

```diff
@@ -35,26 +35,18 @@ pip install virtualenv
 ``` sh
 sudo port install python27 py27-virtualenv cmake
 ```
-#### On OS X 10.11 (El Capitan), you also have to install openssl
+#### On OS X <= 10.11 (El Capitan), you also have to install openssl
```

Comment:
I can't know (have zero access to it), but I'm hopping it will come with a new openssl library.